pkg/api/defaults.go

```diff
@@ -508,6 +512,76 @@ func (p *Properties) setHostedMasterProfileDefaults() {
 	p.HostedMasterProfile.Subnet = DefaultKubernetesMasterSubnet
 }
 
+func (p *Properties) setCustomCloudProfileDefaults() {
```

Comment:
Let's move this method declaration to a new file (like defaults-customcloudprofile.go perhaps?) as `defaults.go` is fairly big as it is.